tensorflow/contrib/learn/python/learn/learn_io/data_feeder.py

```diff
@@ -363,6 +363,11 @@ def __init__(self,
 
     num_samples = list(self._x.values())[0].shape[
```

Comment:
Can you refactor the logic above so that we have all three cases clearly checked?

Reply:
@mdymczyk can you use `is_tensor` as recommended?

Reply:
@drpngx ah yes sorry, this has totally slipped my mind - busy prepping for StrataNY.